tests/processes/ProcessingTest.py

```diff
@@ -11,9 +11,11 @@
 
 from pyprint.ConsolePrinter import ConsolePrinter
 
-from testfixtures import LogCapture, StringComparison
+from testfixtures import LogCapture, StringComparison, TempDirectory
 
 from coalib.bears.Bear import Bear
+from coalib.output.ConsoleInteraction import (acquire_settings,
```

Comment:
Hey, @ishanSrt I think this one is unused import :)

Reply:
hmmm, travis pointed out one unused import to me, I wonder why it didn't point out these ones